packages/react-examples/src/react-charting/AreaChart/AreaChart.Multiple.Example.tsx

```diff
@@ -211,6 +211,7 @@ export class AreaChartMultipleExample extends React.Component<{}, IAreaChartBasi
             legendProps={{
               allowFocusOnLegends: true,
             }}
+            optimizeLargeData
```

Comment:
Add a label that this chart is using optimizeLargeData flag.

Reply:
added separate example for it